roles/common/tasks/main.yml

```diff
@@ -4,8 +4,8 @@
   with_items: "{{ trusted_networks }}"
 - name: Set hostname to fqdn
   hostname: name='{{ inventory_hostname }}'
-#- name: Dont bind hostname to loopback
-#  replace: dest=/etc/hosts regexp='^.*{{ inventory_hostname }}.*$' replace='' backup=yes
+- name: Dont bind hostname to loopback
+  replace: dest=/etc/hosts regexp='^(127.0.0.1).*{{ inventory_hostname }}.*$' replace='' backup=yes
```

Comment:
When I create a fresh Vagrant machine with `vagrant init ubuntu/xenial64` and `vagrant up` the /etc/hosts
file contains an `127.0.1.1	ubuntu-xenial.localdomain	ubuntu-xenial` entry.

The regex should also remove that line.